_docs/arch_load_balanced.md

```diff
@@ -65,14 +64,12 @@ that require the redundancy of multiple compilers.
 
 {write a guide on how to deploy, configure, and use this architecture}
 
-
```

Comment:
These extra spaces between headlines are incredibly helpful when reading the source doc. Please disable this check.